iBrogueCE_iPad/BrogueCode/platformdependent.c

```diff
@@ -43,142 +43,145 @@ brogueScoreEntry scoreBuffer[HIGH_SCORES_COUNT];
 unsigned int glyphToUnicode(enum displayGlyph glyph) {
     if (glyph < 128) return glyph;
 
+    // environment glyphs should show up in TILES or HYBRID, not TEXT
+    // monster glyphs should only show up in TILES, not TEXT or HYBRID
+    // items glyphs show up only in TILES, per isEnvironment()
+    // maybeTiled() is #defined in platform.h
+    
+
+    if (graphicsMode == TILES_GRAPHICS || (graphicsMode == HYBRID_GRAPHICS && isEnvironmentGlyph(glyph))) return (glyph-130)+0x4000;
```

Comment:
In the interest of reducing diffs you may want to do this in the platform side if it's possible, that's what the other platforms do - and looks like there is also a change to the big switch block below

Reply:
That was my original intent, but I ran into some barriers. Since tiles.c was on the C engine side, I thought it was OK to modify the C engine side to use the BrogueCE font, but I know that's not ideal. 

I think I know how to pull it back over to the platform-specific code, so I'll work on that.

I made changes to the large SWITCH, but then a later commit undid those changes, so the true diff is just the lines you highlighted.

Reply:
OK, I rewrote the code on the platform specific side, and reverted platformdependent.c to its former pristine state.

however .. I did make a small change to io.c to move the dialogue windows up 4 lines, so they would be above the iPad onscreen keyboard. Previously, sometimes the keyboard would obscure the Generate Dungeon Seed box.